server/server.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+const cors = require('cors');
+const fetch = require('node-fetch');
+const dotenv = require('dotenv');
+
+dotenv.config();
+const port = process.env.PORT;
+const user = process.env.USER;
+const password = process.env.PASSWORD;
+
+const app = express();
+app.use(express.json());
+app.use(cors());
+
+app.use(function (req, res, next) {
+  res.header('Access-Control-Allow-Origin', '*');
+  res.header(
+    'Access-Control-Allow-Headers',
+    'Origin, X-Requested-With, Content-Type, Accept, Authorization'
+  );
+  res.header('Access-Control-Allow-Methods', 'PUT, POST, GET, OPTIONS');
+  next();
+});
+
+app.post('/auth', async (req, res) => {
+  try {
+    const body = { user: user, password: password, expire_hours: -1 };
```

Comment:
good for now, later please try to use token with expiration period

Reply:
Ok, I'll do later